### DISCUSSION TITLE
2. ROI 추출 (Faster RCNN)

### DISCUSSION THREAD

**Discussion by SHEYCHAE** [kikiru328/Bone_Detection#22] [Study]
# 2. ROI 추출

- 객체 검출에 사용되는 Faster RCNN사용 
- 손목 : Ulna , Radius
- 엄지: Distal Phalanx, Proximal Phalanx, Metacarpal
- 중지: Distal Phalanx, Proximal Phalanx, Middle Phalanx, Metacarpal
- 소지: Distal Phalanx, Proximal Phalanx, Middle Phalanx, Metacarpal

- Faster RCNN 학습을 위한 신경망 구조 VGGNET-16 사용
- X-Ray 영상 기준으로 각 BR과 BR에 포함된 뼈의 영역 좌표 필요
  ( x,y좌표, 너비와높이)
- 각 BR별로 따로 학습 : X-Ray영상 한 장당 4개의 학습모델 생성

**Comment by kikiru328** [comment]
![image](https://user-images.githubusercontent.com/60537388/137865730-ac599089-07d5-4fd1-8044-7a6921904f23.png)
